src/js/pannellum.js

```diff
@@ -350,6 +350,11 @@ function init() {
         };
         
         var onError = function(e) {
+            if (1) { // support partial cubemap image, i.e., missing faces
+                console.log(config.strings.fileAccessError.replace('%s', e.target.src)+'; will use background instead');
+                onLoad();
+                return;
+            }
```

Comment:
This prevents helpful error messages when URLs are incorrect. Missing faces should be defined as `null` in the configuration, and this should then be checked instead of trying and failing to load the face.